laser_scan_topic: '/laser_scan'
odom_topic: '/odom_rf2o' 
publish_tf: false
base_frame_id: '/base_link'
odom_frame_id: '/odom' 
init_pose_from_topic: ''
freq: 10.0
verbose: true